### ISSUE TITLE
[Feature Request]: Support kebab-case for policy methods

### ISSUE THREAD

**Issue by dmitry-ivanov** [laravel-idea/plugin#1112]
### Feature Description

Assuming you have a multi-word policy method _(f.e. `doSomething()`)_, it would be nice to have autocomplete working for the kebab-case as well _(when typing the policy method)_:

```php
// In controller (or Livewire components)
$this->authorize('do-something', $model);

// Via Gate Facade
Gate::authorize('do-something', $model);

// In Blade templates
@can('do-something', $model)

// etc.
```

PS: Currently, it autocompletes only the camelCase _(`'doSomething'`)_.

**Comment by adelf** [comment]
We are going to add it. Here is a poll about what should be done by default: https://x.com/laravel_idea/status/1864641097890189618

**Comment by adelf** [comment]
Implemented in 9.1. Thanks!
